modules/installation-applying-aws-security-groups.adoc

```diff
@@ -0,0 +1,86 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-government-region.adoc
+// * installing/installing_aws/installing-aws-secret-region.adoc
+// * installing/installing_aws/installing-aws-private.adoc
+// * installing/installing_aws/installing-aws-vpc.adoc
+
+ifeval::["{context}" == "installing-aws-localzone"]
+:local-zones:
+endif::[]
+
+:_content-type: PROCEDURE
+[id="installation-aws-vpc-security-groups_{context}"]
+= Applying existing AWS security groups to the cluster
+
+Applying existing AWS security groups to your control plane and compute machines can help you meet the security needs of your organization, in such cases where you need to control the incoming or outgoing traffic of these machines.
+
+.Prerequisites
+* You have created the security groups in AWS. For more information, see the AWS documentation about working with link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-security-groups.html[security groups].
+* The security groups must associated with the existing VPC to which you are deploying the cluster. The security groups cannot be associated with another VPC.
+* You have an existing `install-config.yaml` file.
+
+.Procedure
+
+. Edit the `compute.platform.aws.additionalSecurityGroupIDs` parameter to specify one or more custom security groups for your compute machines.
+. Edit `controlPlane.platform.aws.additionalSecurityGroupIDs` parameter to specify one or more custom security groups for your control plane machines.
+. Save the file and reference it when deploying the cluster.
+
+.Sample `install-config.yaml` files that specifies custom security groups
+[source,yaml]
+----
+#...
+compute:
+- hyperthreading: Enabled
+ifndef::local-zones[]
+  name: worker
+endif::local-zones[]
+ifdef::local-zones[]
+  - name: edge
```

Comment:
```suggestion
compute:
- hyperthreading: Enabled
ifndef::local-zones[]
  name: worker
endif::local-zones[]
ifdef::local-zones[]
  name: edge
```

Reply:
Done. Thank you for the clarification on Friday. I did not know the `-` from `hyperthreading` was inherited.

Reply:
lgtm that part. Thanks